### DISCUSSION TITLE
Sharing camera resource

### DISCUSSION THREAD

**Discussion by Arduhat** [svpcom/wfb-ng#330] [Questions]
Hi, I am a developer that builds a lot of custom drones. I like to run a lot of software on board that often requires access to the camera. I have previously tried to use OpenHD to do this but I cannot access the camera while OpenHD is using it (obviously to transmit it to me at the ground station). I still want to receive my video and telemetry at the ground station obviously. But is there a way to share the camera resource on the air side so that I can run other custom software I've written myself that requires access to the camera/video?

**Comment by svpcom** [comment]
To split RTP video stream to two destinations you need to use something like:
```
gst-launch-1.0 udpsrc port=5600 ! tee name=qq ! queue ! udpsink host=127.0.0.1 port=5602 sync=true qq. ! queue ! udpsink host=127.0.0.1 port=5604 sync=true
```
